core/internal/cltest/cltest.go

```diff
@@ -1481,3 +1482,22 @@ func RandomizeNonce(t *testing.T, s *strpkg.Store) {
 	err := s.DB.Exec(`UPDATE keys SET next_nonce = ?`, n).Error
 	require.NoError(t, err)
 }
+
+func NewConfigDigest(t *testing.T) ocrtypes.ConfigDigest {
```

Comment:
Since it does not return a pointer, would `MakeConfigDigest` be a more idiomatic name?